wifiphisher/deauth.py

```diff
@@ -7,8 +7,8 @@
 
 import threading
 import scapy.layers.dot11 as dot11
-from scapy.all import *
-from constants import *
+import scapy.arch.linux as linux
+import wifiphisher.constants as constants
```

Comment:
>The naming convention is that constants are in uppercase; ALL_CAPS_CONSTANTS are instantly recognizable that way.

How is `constants.CONSTANT` any less recognizable than `CONSTANT`?

>And if I use constants.ALL_CAPS_CONSTANTS, someone is still allowed to create a ALL_CAPS_CONSTANTS in the file and we don't want that.

Imagine you needed to name a variable `ALL_CAPS_CONSTANT` for some reason. You could still use `constants.ALL_CAPS_CONSTANT` without any problem or loss of clarity.